### PR TITLE
[Network] kRO Sakray Connection

### PR DESCRIPTION
**Pull request by alisonrag**
Add kRO Sakray again to the server list (now working).

to work you will need to use with this plugin:
https://github.com/alisonrag/Openkore-Plugins-Scripts/tree/master/SakrayAuth

### CODE REVIEW

**Comment by alisonrag** [comment]
This is for kro sakray (test server) everything new come first in this server.
kRO have 3 servers:
Main 
Zero
Sakray

**Comment by alisonrag** [comment]
Zero use same authentication, but have packet encryption in map server

File: tables/kRO/Sakray/shuffle.txt [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# kRO Zero Date: 2018-07-18
```
Comment (sctnightcore) [inline tables/kRO/Sakray/shuffle.txt]: kRO Zero ?

File: tables/kRO/Sakray/sync.txt [hunk 2]
```diff
@@ -0,0 +1,86 @@
+# kRO Zero Date: 2018-03-14n
```
Comment (sctnightcore) [inline tables/kRO/Sakray/sync.txt]: kRO Zero ?

**Comment by alisonrag** [comment]
@sctnightcore 
updated

**Comment by Nipodemos** [comment]
anyone can test this?

**Comment by alisonrag** [comment]
have to use the plugin and proxy/vpn kRO block international ip

**Pull request merged**